### ISSUE TITLE
Instead of posting Slack Notification with Emoji or Uploaded Image, send Pantheon Logo in URL

### ISSUE THREAD

**Issue by jordan8037310** [pantheon-systems/quicksilver-examples#100]
Slack accepts full URLs for emoji icons in the slack attachments array, why not enrich the out of box experience with some Pantheon branding? Default can be set to a Pantheon logo URL instead of `:lightning_cloud:`

Ideal would be 128x128px off a CDN.

#foodforthought

**Comment by spiderwisp** [comment]
You can add a custom emoji to Slack. Perhaps drop a 'slack_emoji' field in your $defaults array and use that if set to really personalize.

**Comment by jordan8037310** [comment]
That is true, and that is certainly what I have done for myself @spiderwisp. But you could just as easily post `https://pbs.twimg.com/profile_images/685242046488051712/P2VERqqR_400x400.png` for the Pantheon logo and have it work for everyone out of box.
